test/surface_test.py

```diff
@@ -3766,6 +3766,82 @@ def test_fill_blend_rgba(self):
                 dst.fill(fill_color, special_flags=getattr(pygame, blend_name))
                 self._assert_surface(dst, p, f", {blend_name}")
 
+    def test_surface_premul_alpha(self):
+        """Ensure that .premul_alpha() works correctly"""
+        pygame.display.init()
+        try:
+            pygame.display.set_mode((640, 480))
```

Comment:
Why set up a display and quit it for this test? I haven't seen anything yet that suggests a display is necessary to do this operation.

Reply:
I had no idea, when I first looked at it - then I remembered that this functionality was once bolted on to `surface.convert_alpha()` which does look at the display surface and the test was clearly originally cloned from that functions test and not correctly updated once it became its own thing.

I have done so now, thanks!